src/sidebar/components/Annotation/AnnotationUser.js

```diff
@@ -17,9 +17,9 @@ function AnnotationUser({ authorLink, displayName }) {
 
   if (authorLink) {
     return (
-      <Link href={authorLink} target="_blank">
+      <LinkBase href={authorLink} target="_blank">
```

Comment:
This uses `LinkBase` instead of link because styling is provided by the `h3` utility classes and the resulting link does not correspond to any standard `Link` colors.